package.json

```diff
@@ -27,19 +27,19 @@
     "postinstall": "typings install"
   },
   "dependencies": {
-    "qrcodejs2": "^0.0.2",
-    "@angular/common": "2.0.0-rc.1",
-    "@angular/compiler": "2.0.0-rc.1",
-    "@angular/core": "2.0.0-rc.1",
-    "@angular/platform-browser": "2.0.0-rc.1",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.1",
+    "qrcodejs2": "^0.0.2"
+  },
+  "peerDependencies": {
+    "@angular/core": "^2.0.0-rc.4",
```

Comment:
We only `need angular/core`. I also could lower the version number back to rc.1.